tomopy/recon/algorithm.py

```diff
@@ -140,6 +140,12 @@ def recon(
             Parzen filter.
         'butterworth'
             Butterworth filter.
+        'custom'
```

Comment:
It would be nice to explain if DC term is at the beginning of the array or at the middle? (e.g. if ```fftshift``` is needed or not for the filters.)